media/docs/cpp/cute/02_layout_algebra.md

```diff
@@ -147,9 +151,15 @@ Put into words, `A o B = A o s:d`, for integral `s` and `d` means that we want (
 * `(3,6,2,8) /  9 => (1,2,2,8)`
 * `(3,6,2,8) / 72 => (1,1,1,4)`
 
+To understand how this allows us to calculate the stride of the composition, consider for the composition `(6,2):(x, y) o (2, 2)`, where `x, y` are integers.
+
+"Dividing out" to find the shape of view we see that `(6,2) / 2 = (3,2)`, implying that the stride of the composed layouts is `(2*x, y)` - incorporating both strides.
+
 As you may have noticed, we can only divide shapes by certain values and get a sensible result. This is called the **stride divisibility condition** and is statically checked in CuTe when possible.
 
-2. Keeping the first `s` elements of the strided `A` layout can be computed by "modding out" the first `s` elements from the shape of `A` starting from the left. For example,
+2. Keep the first `s` elements of the newly strided `A`. This defines the shape of the composed layout. This can be computed by "modding out" the first `s` elements from the shape of `A` starting from the left.
```

Comment:
```suggestion
2. Keep the first `s` elements of the newly strided `A` so that the result has a compatible shape with `B`. This can be computed by "modding out" the first `s` elements from the shape of `A` starting from the left.
```